nl/guides/students/exercises/README.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Oefeningen oplossen"
+title: "[nl]Oefeningen oplossen"
```

Comment:
Since this page is in Dutch and on the Dutch part of the website, the nl tag shouldn't be here.